.lighthouserc.js

```diff
@@ -0,0 +1,232 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
@vojtechjelinek any reason why this is moved to a javascript file?

Reply:
I believe its to be able to have comments in the config

Reply:
Okay, cool.